backend/tests/test_validation.py

```diff
@@ -34,19 +32,17 @@ def test_invalid_location_too_many_fields():
         'extra': 'peilun'
     }
     try:
-        validation.validate_location(loc)
-        assert False
-    except:
-        pass
+        validate_location(loc)
```

Comment:
All invalid input test cases should fail if validate() does not throw

Reply:
Corrected